frontend/main.css

```diff
@@ -0,0 +1,22 @@
+#map {
+	height: 70vh;
+}
+
+.spinner-loader {
+	border: 5px solid #DEE0E2;
+	border-radius: 50%;
+	border-top-color: #005EA5;
+	width: 30px;
+	height: 30px;
+	animation: spin 1s linear infinite;
```

Comment:
oh fancy, nice use of keyframes
For a review you could also grab a screenshot of gif of that and stick it on the PR.

This looks cool but I can't visualise what it looks like without running the code otherwise!

Reply:
Done, added to PR! Not sure if there's a way of adding media to a PR inline, or if the way I've done it is best...